lib/jwt.go

```diff
@@ -0,0 +1,87 @@
+package lib
+
+import (
+	"errors"
+	"net/http"
+	"strings"
+	"time"
+
+	"github.com/dgrijalva/jwt-go"
+)
+
+type Claims struct {
+	Username string `json:"username"`
+	jwt.StandardClaims
+}
+
+var jwtKey = []byte("my_secret_key") //Todo hide key
```

Comment:
// Todo do wykonania :D

Reply:
Tutaj pytanko, czy bardziej zabawić się w pakiecie viper do pliku konfiguracyjnego, czy dodać zmienną środowiskową, jak w przypadku np. bazy danych?

Reply:
Możesz dać to do enva, nie ma sensu bawić się raczej w config dla jednej wartości a i tak bazę trzymasz w envie.